samples/app/src/main/java/com/example/app/env/EnvViewModel.kt

```diff
@@ -46,9 +44,9 @@ class EnvViewModel : ViewModel() {
 
     val dbind = FROptionsBuilder.build {
         server {
-            url = "https://openam-updbind.forgeblocks.com/am"
-            realm = "bravo"
-            cookieName = "ccdd0582e7262db"
+            url = "http://192.168.86.32:8080/openam"
```

Comment:
Do we want to commit this change?

Reply:
Yes, this file is more for testing.
Environment keep changing.
We may eventually use a valid environment, so that developer can test with our environment, and add this to .ignore file.
We can discuss about the approach later, but for now, this is a file for internal testing.